CanvasCore/CanvasCore/WhizzyWig/WhizzyWigTableViewCell.swift

```diff
@@ -31,7 +31,7 @@ open class WhizzyWigTableViewCell: UITableViewCell {
         }
     }
     
-    @objc var minHeight: CGFloat = 0.0
+    @objc var minHeight: CGFloat = 80.0
```

Comment:
We have plans to rewrite the quiz details screen. So this is just a band aid until we can get around to that.